lib/actions/preload.coffee

```diff
@@ -100,16 +100,18 @@ offerToDisableAutomaticUpdates = (application, commit) ->
 			body:
 				should_track_latest_release: false
 
+progressBars = {}
```

Comment:
Is there a reason this is defined outside the action, but `spinners` is indeed? Personally, I'd put both inside, just to make it clearer what this relates to.

Reply:
There's no reason to keep it here, I just forgot to move it, it is fixed now.